libs/ob-sql-parser/src/test/java/com/oceanbase/tools/sqlparser/adapter/OraclePartitionFactoryTest.java

```diff
@@ -599,23 +599,9 @@ public void generate_listPartitionWithSubPartitionOptionsQuantity_succeed() {
         Assert.assertEquals(expect, actual);
     }
 
-    @Test
-    public void generate_columnPartition_succeed() {
```

Comment:
there is no 'column partition' indeed, the case is for cover previous g4 syntax definition